daily-challenges/2024-09/20/lcaloi_29.spec.ts

```diff
@@ -0,0 +1,40 @@
+//Javascript
+import {test, expect, Locator} from "@playwright/test";
```

Comment:
VD file này chưa format này.
Nếu format rồi thì sẽ là
```suggestion
import { test, expect, Locator } from "@playwright/test";
```

Reply:
> VD file này chưa format này. Nếu format rồi thì sẽ là
à à. okei anh. để em đi từng file format lại